code/game/objects/items.dm

```diff
@@ -340,6 +340,8 @@ var/global/image/fire_overlay = image("icon" = 'icons/goonstation/effects/fire.d
 		A.Remove(user)
 	if(flags & DROPDEL)
 		qdel(src)
+	if((flags & NODROP) && !(initial(flags) & NODROP)) //Remove NODROP is dropped
+		flags &= ~NODROP

```

Comment:
this will probabbbbblyyy break things.

Reply:
Most antidrop items either have it set in their construction. On pickup or by admemery. 
I know the last one will break.
If it's set when the person is holding it in his hands then it should be removed after it's gone from it's hands and reset once somebody picks it up.

It will break stuff but that stuff is probably poorly made. Fixing them would be cleaner than making more edge cases for antidrop (on surgery removal, on gib.. etc)